javascript/javascript1/week3/homework4.js

```diff
@@ -0,0 +1,74 @@
+// Smart-ease - Back to the basics!
+// NOnoN0nOYes (Note taking app)
+
+// The saveNote function should push an object to the notes array with the keys content and id.
+
+const notes = [];
+/**
+ *
+ *
+ * @param {string} content
+ * @param {number} id
+ */
+
+function saveNote(content, id) {
+  notes.push({
+    content: content,
+    id: id,
+  });
```

Comment:
I think it also works like this:
```suggestion
  notes.push({ content, id});
```